app/matplotlib.ipynb

```diff
@@ -86,23 +86,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
     "x = np.random.rand(50)\n",
     "y = np.random.rand(50)\n",
     "colors = np.random.rand(50)\n",
     "sizes = 100 * np.random.rand(50)\n",
     "\n",
-    "# write your code here"
+    "plt.scatter(x, y, sizes, colors)\n",
+    "plt.title(\"Scatter\")\n",
+    "plt.ylabel(\"entry Y\")\n",
+    "plt.xlabel(\"entry X\")\n",
+    "plt.show()"
```

Comment:
The scatter plot is missing a colorbar label. You should add a colorbar and set its label to fulfill the task requirement.